util/flags.py

```diff
@@ -23,6 +23,7 @@ def create_flags():
     # ================
 
     tf.app.flags.DEFINE_boolean ('train',            True,        'whether to train the network')
+    tf.app.flags.DEFINE_boolean ('dev',              True,        'whether to run validation epochs')
```

Comment:
I already thought a couple of times about dropping these flags in favour of checking, if the according data set has been provided.

Reply:
Ah, that's a better idea, I'll change it to check for the files.